scripts/exglobal_atmos_tropcy_qc_reloc.sh

```diff
@@ -93,27 +91,27 @@ if [ "$DO_RELOCATE" = 'YES' ]; then
 ###################################################
 
    export MP_LABELIO=${MP_LABELIO:-yes}
-   ${USHgfs}/tropcy_relocate.sh $cdate10
+   "${USHgfs}/tropcy_relocate.sh" "${cdate10}"
    errsc=$?
 
-   [ "$errsc" -ne '0' ]  &&  exit $errsc
+   [[ "${errsc}" -ne '0' ]]  &&  exit ${errsc}
 
 
 # save global sigma guess file(s) possibly updated by tropical cyclone
 #  relocation processing in COMSP path
-   qual_last=".$tmmark"  # need this because gfs and gdas don't add $tmmark
+   qual_last=".${tmmark}"  # need this because gfs and gdas don't add $tmmark
                          #  qualifer to end of output sigma guess files
-   [ $RUN = gfs -o $RUN = gdas -o $NET = cfs ]  &&  qual_last=""
+   [ "${RUN}" = gfs -o "${RUN}" = gdas -o "${NET}" = cfs ]  &&  qual_last=""
 
-   if [ $BKGFREQ -eq 1 ]; then
+   if [[ "${BKGFREQ}" -eq 1 ]]; then
       if [[ -s sgm3prep ]]; then cp "sgm3prep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgm3prep${qual_last}"; fi
       if [[ -s sgm2prep ]]; then cp "sgm2prep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgm2prep${qual_last}"; fi
       if [[ -s sgm1prep ]]; then cp "sgm1prep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgm1prep${qual_last}"; fi
       if [[ -s sgesprep ]]; then cp "sgesprep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgesprep${qual_last}"; fi
       if [[ -s sgp1prep ]]; then cp "sgp1prep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgp1prep${qual_last}"; fi
       if [[ -s sgp2prep ]]; then cp "sgp2prep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgp2prep${qual_last}"; fi
       if [[ -s sgp3prep ]]; then cp "sgp3prep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgp3prep${qual_last}"; fi
-   elif [ $BKGFREQ -eq 3 ]; then
+   elif [[ "${BKGFREQ}" -eq 3 ]]; then
```

Comment:
No need for quotes when checking via `-eq`, `-ge`, etc.
```suggestion
   if [[ ${BKGFREQ} -eq 1 ]]; then
      if [[ -s sgm3prep ]]; then cp "sgm3prep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgm3prep${qual_last}"; fi
      if [[ -s sgm2prep ]]; then cp "sgm2prep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgm2prep${qual_last}"; fi
      if [[ -s sgm1prep ]]; then cp "sgm1prep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgm1prep${qual_last}"; fi
      if [[ -s sgesprep ]]; then cp "sgesprep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgesprep${qual_last}"; fi
      if [[ -s sgp1prep ]]; then cp "sgp1prep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgp1prep${qual_last}"; fi
      if [[ -s sgp2prep ]]; then cp "sgp2prep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgp2prep${qual_last}"; fi
      if [[ -s sgp3prep ]]; then cp "sgp3prep" "${COMOUT_OBS}/${RUN}.t${cyc}z.sgp3prep${qual_last}"; fi
   elif [[ ${BKGFREQ} -eq 3 ]]; then
```

Reply:
Thanks for the various edits @DavidHuber-NOAA ! I have incorporated them in my branch.